etc/yafti.yml

```diff
@@ -21,9 +21,6 @@ screens:
         - run: /usr/lib/fedora-third-party/fedora-third-party-opt-out
         - run: /usr/bin/fedora-third-party disable
         - run: flatpak remote-delete fedora --force
-        - run: flatpak remove --system --noninteractive --all
```

Comment:
Curious - how do you plan to remove the pre-installed Fedora Flatpaks when rebasing from an existing image?  
Or are you assuming this will be installed via ISOs - therefore not contain existing Flatpaks?

Reply:
@p5 yeah I wondered about this as well; doesn't the line immediately above handle that?

But it should definitely be tested. I didn't remove this line in endlish-oesque (https://github.com/ublue-os/endlish-oesque/blob/36f22545c9e1cac3cafb45848fb022067b21c9a1/etc/yafti.yml#L20-L26) and I recall some issues with the system remote being removed and re-added unnecessarily.

Reply:
I could copy this back into endlish-oesque and just see what happens when rebasing from a stock Fedora image, as it's an even more experimental image and I don't mind pushing broken things there. 😄

Reply:
I'm trying to update your script to add logic to remove Fedora flatpaks if the oneshot hasn't been ran already.  Will let you know if it works
https://github.com/rsturla/eternal-images/pull/21/files#diff-cac88664baf2bb1d4c843b4247ea9ab0ccaa0b13b00cf55a5c63768f0f4dd97eR1

Reply:
@p5 ah you're right; I wasn't aware a Flatpak could get orphaned like that. I'll poke around a bit with the Flatpak CLI and docs. 😄

Reply:
I mean rather than removing all, I could just specifically target the Fedora remote. Duh! 🤦🏻

Reply:
Has this method of installing Flatpaks been confirmed in Silverblue OCI images?
It looks like files are stored in `/var/lib/`, which is not allowed by ostree.  

```
98.73 Found file: var/lib/flatpak/repo/objects/4b/54aceab94ad094bbc1154f2f160a77b27c3f339caf57c1cfb7da33b9ce534d.file
98.73 Found file: var/lib/flatpak/repo/objects/4b/b51a1d2fb4dbb5eda84f52cd47862efbb4c62f03d27512e3f6c66c3df8c0de.file
98.91 error: Found content in var
```

Where does Flatpak store the artefacts from `--no-deploy`?

Reply:
Ohh, you can change the cache directory using the following environment variable
```
FLATPAK_SYSTEM_CACHE_DIR
```
https://docs.flatpak.org/en/latest/flatpak-command-reference.html#Environment

We will likely need to set that to somewhere else since it defaults to `/var/tmp`, which must be clear when running `ostree container commit`

Reply:
I think it'd be fine to add this line to the Containerfile (or a script called by it) so that they get removed while the image is built